scss/titles.scss

```diff
@@ -0,0 +1,19 @@
+.title {
+  color: #000;  
```

Comment:
use color variables

Reply:
For this, create a `colors.scss` file and create colors like `$black: #000;`
Then, add `@import 'colors';`  in application.scss. And then you can use `color: $black`